docs/our-approach-to-data.md

```diff
@@ -332,9 +332,8 @@ export const items = createReducer( defaultState, {
 	[DESERIALIZE]: state => fromJS( state )
 } );
 ```
-If your reducer state is already a plain object, you may choose to omit `SERIALIZE` and `DESERIALIZE` handlers, or 
-simply define them as returning the current state. However, please note that the subtree can still see errors from 
-changing data shapes, as described below.
+If your reducer state is already a plain object, the `SERIALIZE` and `DESERIALIZE` handlers are not needed.
+However, please note that the subtree can still see errors from changing data shapes, as described below.
```

Comment:
Ah I should probably reword the section to say plain object, string or bool. For simple types like string or boolean it's harder to get into trouble, but we should have a schema for it anyway since it's still possible to run into data shape problems. The most common example might be changing a prop from a boolean to a string, to allow for more than two states.